test/replication.test.js

```diff
@@ -52,6 +52,121 @@ describe('Replication / Change APIs', function() {
     };
   });
 
+  describe('optimization check rectifyChange Vs rectifyAllChanges', function() {
+    beforeEach(function initialData(done) {
+      var data = [{name: 'foo1', lname: 'bar1'}, {name: 'foo2', lname: 'bar2'}];
+      async.auto({
```

Comment:
I am out of the office until 01/04/2016.

Happy holidays!


Note: This is an automated response to your message  "Re: [loopback] WIP:
Rectify change vs rectify all changes [Optimization] (#1899)" sent on
12/21/15 9:27:13 AM.

This is the only notification you will receive while this person is away.